src/ngraph/pass/pretty_print.cpp

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
```

Comment:
We already have a pretty print pass, it is called dump_sorted. We don't need two passes that do the same thing so merge them together or delete one.